downstream/modules/platform/con-aap-installation-on-disconnected-rhel.adoc

```diff
@@ -1,34 +1,34 @@
-
+// Module included in the following assemblies:
+// assembly-disconnected-installation.adoc
 
 [id="con-aap-installation-on-disconnected-rhel_{context}"]
 
-
-= {PlatformNameShort} installation on disconnected RHEL
+= Installing {PlatformNameShort} on disconnected RHEL
 
 [role="_abstract"]
-Install Ansible Automation Platform (AAP) automation controller and a private automation hub, with an installer-managed database located on the automation controller without an Internet connection.
+You can install {PlatformNameShort} {ControllerName} and a {PrivateHubName} without an internet connection using the installer-managed database located on the {ControllerName}.
 
 == Prerequisites
```

Comment:
Couple of things: 

- Great job on removing the chapter level attribute from this prerequisite section. The only thing is prerequisites usually included in procedure or task modules only. Also, the syntax is usually `.Procedure` so that they display within a chapter and not in a sub-topic on its own. Let me know if this is confusing.